lib/sqlalchemy/dialects/postgresql/asyncpg.py

```diff
@@ -98,6 +98,44 @@
    stale, nor can it retry the statement as the PostgreSQL transaction is
    invalidated when these errors occur.
 
+.. _asyncpg_prepared_statement_name:
+
+Prepared Statement Name
+-----------------------
+
+By default, asyncpg enumerates prepared statements in numeric order, which 
```

Comment:
**mike bayer** (zzzeek) wrote:

this is great!  nice job, well written

want some more doc tasks ?  :)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4565

Reply:
hah thanks, well I might help if find the time :D